Docs: improve contributing sign posting (#2888)
ArchipelagoMW/Archipelago: docs/contributing.md

  Please also tell us if you looked at code, just did functional testing, did both, or did neither.
  If testing the PR depends on other PRs, please state what you merged into what for testing.
  We cannot determine what "LGTM" means without additional context, so that should not be the norm.

Other than these requests, we tend to judge code on a case-by-case basis.

For contribution to the website, please refer to the [WebHost README](/WebHostLib/README.md).

If you want to contribute to the core, you will be subject to stricter review on your pull requests. It is recommended
that you get in touch with other core maintainers via the [Discord](https://archipelago.gg/discord).

If you want to add Archipelago support for a new game, please take a look at
the [adding games documentation](/docs/adding%20games.md)
which details what is required to implement support for a game, and has tips on to get started.
If you want to merge a new game into the main Archipelago repo, please make sure to read the responsibilities as a
[world maintainer](/docs/world%20maintainer.md).

For other questions, feel free to explore the [main documentation folder](/docs), and ask us questions in the

> #archipelago-dev channel of the [Discord](https://archipelago.gg/discord).

Reviewer: ```suggestion
#ap-world-dev channel of the [Discord](https://archipelago.gg/discord).
```

# Contributing
Contributions are welcome.

> We have a few asks of contributors, whether they be for core, webhost, or new game contributions:

Reviewer: I think "asks of" is quite formal, to the point that it's almost awkward. "requests for" or even "requirements for" would both be better imho

If you want to contribute to the core, you will be subject to stricter review on your pull requests. It is recommended
that you get in touch with other core maintainers via the [Discord](https://archipelago.gg/discord).

If you want to add Archipelago support for a new game, please take a look at the [adding games documentation](/docs/adding%20games.md),

> which details what is required to implement support for a game, as well as tips for how to get started.

Reviewer: The comma at least should be removed here, but I think "details [X] as well as tips" is a bit awkward, so this might be better.
It also should either be "tips *on* how to get started" or "tips *for* getting started"
```suggestion
which details what is required to implement support for a game and has tips on how to get started.
```
